test/fast.js

```diff
@@ -157,6 +157,46 @@ describe('aws4', function() {
     })
   })
 
+  describe('#uriEscape()', function() {
+    var uriEscape = function (str) {
```

Comment:
I'm not sure I really understand – you're redeclaring the function here and then testing it? This won't actually test the function declared in `aws4.js`